playground/components/deeper-sfc.vue

```
@@ -0,0 +1,13 @@
+<script setup lang="ts">
+  import { z } from 'zod'
+
+  const schema = z.object({ fruit: z.string() })
+  const { register } = useForm({ schema, key: 'example-form-1' })
```

ESLint

[error] 'useForm' is not defined.

(no-undef)